homeassistant/components/drop/config_flow.py

```diff
@@ -0,0 +1,152 @@
+"""Config flow for DROP."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.service_info.mqtt import MqttServiceInfo
+from homeassistant.util.json import JSON_DECODE_EXCEPTIONS, json_loads
+
+from .const import (
+    CONF_COMMAND_TOPIC,
+    CONF_DATA_TOPIC,
+    CONF_DEVICE_DESC,
+    CONF_DEVICE_ID,
+    CONF_DEVICE_NAME,
+    CONF_DEVICE_TYPE,
+    CONF_HUB_ID,
+    CONF_UNIQUE_ID,
+    DISCOVERY_TOPIC,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
I actually started with the logger defined in const.py but later moved it to each module so that log entries include the module name, which I found helpful.  A cursory survey of other integrations shows that this approach is very common, so I'd like to leave it as-is if possible.